piet-coregraphics/src/text.rs

```diff
@@ -134,12 +134,19 @@ impl CoreGraphicsTextLayoutBuilder {
                         color.as_CFType(),
                     )
                 }
-                TextAttribute::Underline => {
-                    #[allow(non_upper_case_globals)]
+                #[allow(non_upper_case_globals)]
```

Comment:
Do you have to move this to avoid the warnings?

Reply:
previously with a single `const` here I could put this annotation on the individual item, but adding a second `const` requires me to put it on the nearest containing block.